enhancements/windows-containers/monitoring-windows-nodes.md

```diff
@@ -0,0 +1,144 @@
+---
+title: monitoring-windows-nodes
+authors:
+  - "@VaishnaviHire"
+  - "@ravisantoshgudimetla"
+reviewers:
+  - "@@openshift/openshift-team-windows-containers"
+approvers:
+  - "@sdodson"
+creation-date: 2021-02-08
+last-updated: 2021-02-11
+status: implementable
+---
+
+# Monitoring Windows Nodes
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The intent of this enhancement is to enable monitoring on Windows nodes created
+by Windows Machine Config Operator(WMCO) in OpenShift cluster.
+
+## Motivation
+
+Monitoring is critical to identify issues with nodes, containers running on the
+nodes. The main motivation behind this enhancement is to enable monitoring on
+the Windows nodes.
+
+### Goals
+
+As part of this enhancement, we plan to do the following:
+* Run Windows exporter as a service on Windows nodes
+* Upgrade Windows exporter on the Windows Nodes
+* Leverage cluster-monitoring operator that sets up Prometheus, Alertmanager
+  and other components
+
+### Non-Goals
+
+As part of this enhancement, we do not plan to do the following:
+* Integrating Windows exporter with cluster monitoring operator
+* Ship Grafana dashboards for Windows Nodes
+
+## Proposal
+
+The main idea here is to run Windows exporter as a Windows Service and let 
+Prometheus instance which was provisioned as part of OpenShift install to 
+collect data from Windows exporter. The metrics exposed by the Windows exporter 
+will be used to display node graphs and workload graphs for Windows nodes. 
+
+## Justification
+
+Unlike node exporter on linux nodes, Windows exporter cannot run as a container 
+on the Windows nodes since Windows container images contains a Windows Kernel 
+and Red Hat has a policy not to ship third party kernels for support reasons.
+Please refer [WMCO enhancement](https://github.com/openshift/enhancements/blob/master/enhancements/windows-containers/windows-machine-config-operator.md#justification).
+Hence, we need to ensure that Prometheus operator watches over the Windows
+exporter external service.
```

Comment:
What is the `Windows exporter external service`?

Reply:
The external service here would be the target exposed by `Windows exporter` : `<internal-ip>:9182/metrics`

Reply:
That is not immediately obvious by referring to it as `Windows exporter external service`. I suggest explaining it.